runescape-client/src/main/java/Item.java

```diff
@@ -36,16 +36,17 @@
       signature = "(LPacketBuffer;B)V",
       garbageValue = "74"
    )
-   static final void method1665(PacketBuffer var0) {
-      for(int var1 = 0; var1 < class96.field1493; ++var1) {
-         int var2 = class96.field1500[var1];
-         Player var3 = Client.cachedPlayers[var2];
-         int var4 = var0.readUnsignedByte();
-         if((var4 & 1) != 0) {
-            var4 += var0.readUnsignedByte() << 8;
+   @Export("updatePlayersFlag")
+   static final void updatePlayersFlag(PacketBuffer packetBuffer) {
```

Comment:
Information about how this works can be found [here](http://rsps.wikia.com/wiki/317_Protocol#Player_Updating) under "Update block flag-based updating". Suggestions for a better name are welcome.